hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdiff/TestRocksDBCheckpointDiffer.java

```diff
@@ -282,22 +282,22 @@ public void testGetSSTDiffListWithoutDB(String description,
         // Snapshot 0
         + "S 1000 df6410c7-151b-4e90-870e-5ef12875acd5 " + createdTime + " \n"
         // Additional "compaction" to trigger and test early exit condition
-        + "C 000001,000002:000062\n"
+        + "C 1 000001,000002:000062\n"
         // Snapshot 1
         + "S 3008 ef6410c7-151b-4e90-870e-5ef12875acd5 " + createdTime + " \n"
         // Regular compaction
-        + "C 000068,000062:000069\n"
+        + "C 2 000068,000062:000069\n"
         // Trivial move
-        + "C 000071,000064,000060,000052:000071,000064,000060,000052\n"
-        + "C 000073,000066:000074\n"
-        + "C 000082,000076,000069:000083\n"
-        + "C 000087,000080,000074:000088\n"
+        + "C 3 000071,000064,000060,000052:000071,000064,000060,000052\n"
+        + "C 4 000073,000066:000074\n"
+        + "C 5 000082,000076,000069:000083\n"
+        + "C 6 000087,000080,000074:000088\n"
         // Deletion?
-        + "C 000093,000090,000083:\n"
+        + "C 7 000093,000090,000083:\n"
         // Snapshot 2
         + "S 14980 e7ad72f8-52df-4430-93f6-0ee91d4a47fd " + createdTime + "\n"
-        + "C 000098,000096,000085,000078,000071,000064,000060,000052:000099\n"
-        + "C 000105,000095,000088:000107\n"
+        + "C 8 000098,000096,000085,000078,000071,000064,000060,000052:000099\n"
+        + "C 9 000105,000095,000088:000107\n"
         // Snapshot 3
         + "S 17975 4f084f6e-ed3d-4780-8362-f832303309ea " + createdTime + "\n";
```

Comment:
Should be in monotonically increasing order considering the "S " entries? e.g.

```
S 1000
C 2000
S 3008
...
```

Reply:
No, it doesn't have to be. Updated it.